File: Zad2/app/src/main/java/com/example/bmizad2/ui/notifications/PPEViewModel.java
package com.example.bmizad2.ui.notifications;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class PPEViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public PPEViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Count your basic metabolism:");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: Zad4/app/src/main/java/com/example/bmizad2/ui/notifications/PPEViewModel.kt
package com.example.bmizad2.ui.notifications

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class PPEViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()

    init {
        mText.value = "Count your basic metabolism:"
    }

    val text: LiveData<String>
        get() = mText
}